### PR TITLE
feat: improve mobile editing experience

### PR DESCRIPTION
**Pull request by jacob-ebey**
Makes side menu collapse for mobile

### CODE REVIEW

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Sounds good to me! Could we swap `classnames` for [`clsx`](https://npm.im/clsx)? It's smaller/faster/stronger/etc.

File: app/routes/notes.tsx [hunk 1]
```diff
@@ -36,35 +54,55 @@ export default function NotesPage() {
...
-            </ol>
+        <div
+          className={cn(
+            "h-full overflow-x-hidden border-r bg-gray-50 md:w-80 md:min-w-[20rem]",
+            "transition-all duration-150 ease-out",
+            {
+              "w-80 min-w-[20rem]": menuOpen,
+              "w-0 min-w-0": !menuOpen,
+            }
           )}
```
Comment (MichaelDeBoey) [inline app/routes/notes.tsx:73]: Do we really need to use `classnames`?

Can't we just do it manually?

```suggestion
          className={`h-full overflow-x-hidden border-r bg-gray-50 transition-all duration-150 ease-out md:w-80 md:min-w-[20rem] ${
            menuOpen ? "w-80 min-w-[20rem]" : "w-0 min-w-0"
          }`}
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: app/routes/notes.tsx (hunk 1 above)
Comment (kentcdodds) [reply app/routes/notes.tsx:73]: We can, but I think most people who are going to keep tailwind around will eventually want to use clsx eventually. It's pretty darn useful.

**Comment by kentcdodds** [comment]
I don't think that built-in mobile support is important here because people are going to delete these files before working on their stuff anyway.

**Pull request closed without merging**